src/mugs.js

```diff
@@ -480,7 +480,7 @@ define([
             return this.parentMug && this.parentMug.isInRepeat();
         },
         supportsRichText: function() {
-            return this.options.richText && this.form.useRichText !== false;
+            return this.options.richText && this.form.noRichText === false;
```

Comment:
Its possible that when loading an empty form it won't have a value for noRichText. You should be able to fix that by initializing it in `Form`. and then you would be able to get rid of the strict checks